### PR TITLE
Add notifications for less and webpack, colors for npm-run-all

### PR DESCRIPTION
**Pull request by k88hudson**
I did a few patches on some of our dev deps that are waiting to land, i thought it would be cool to try them out here:

* autoless: adds notification support and sounds on errors
* webpack-notifier: adds support for per-event custom configuration (error, warning, build, rebuild)
* npm-run-all: preserves colors in `stdout`

### CODE REVIEW

**Comment by thisandagain** [comment]
Nice! Particularly appreciate the `npm-run-all` update.

File: package.json [hunk 1]
```diff
@@ -31,8 +31,8 @@
   },
   "homepage": "https://github.com/mozilla/webmaker-android",
   "devDependencies": {
-    "autoless": "^0.1.7",
     "babel": "^5.3.3",
+    "autoless": "git://github.com/k88hudson/autoless.git#notify",
```
Comment (Pomax) [inline package.json]: Is the thing thing is forked for also filed as PR upstream so we can go back to the next version of autoless in the future? (also a small nit: preserve alphabetical ordering?)

File: package.json [hunk 2]
```diff
@@ -47,14 +47,15 @@
     "mofo-style": "latest",
     "ncp": "^2.0.0",
     "npm": "^2.7.4",
-    "npm-run-all": "^1.2.2",
+    "npm-run-all": "git://github.com/k88hudson/npm-run-all.git#colors",
```
Comment (Pomax) [inline package.json]: same question here; is there a PR to land this for npm-run all?

File: package.json [hunk 3]
```diff
@@ -47,14 +47,15 @@
...
     "parallelshell": "^1.1.1",
     "proxyquire": "^1.4.0",
     "rimraf": "^2.3.2",
     "should": "^5.2.0",
     "tmp": "0.0.25",
     "watch": "^0.14.0",
-    "webpack": "^1.7.3"
+    "webpack": "^1.7.3",
+    "webpack-notifier": "git://github.com/k88hudson/webpack-notifier.git#more-options"
```
Comment (Pomax) [inline package.json]: same question here; is there a PR to land this for webpack-notifier?

File: package.json (hunk 1 above)
Comment (k88hudson) [inline package.json]: Actually, this has now been merged ^_^

File: package.json (hunk 2 above)
Comment (k88hudson) [inline package.json]: Still working on getting this one to work but haven't had time, which is why this PR has been sitting around

**Pull request closed without merging**